gunicorn/config.py

```diff
@@ -177,6 +171,18 @@ def logger_class(self):
         return logger_class
 
     @property
+    def instrumentation_classes(self):
+        inst_class_uris = []
+        inst_classes = []
+        if 'statsd_host' in self.settings and self.settings['statsd_host'].value is not None:
+            inst_class_uris.append("gunicorn.instrument.statsd.Statsd")
+
+        for uri in inst_class_uris:
+            inst_classes.append(util.load_class(uri))
```

Comment:
Probably not a big deal, and you'll have to update your test to use:

    c.instrumentation_classes.next()

instead of your current:

    c.instrumentation_classes[0]

but this feels like it should be an iterator to me, instead of collecting arrays.  Something like:

```python
if 'statsd_host' in self.settings and self.settings['statsd_host'].value is not None:
    yield util.load_class(uri)
```

would cover your whole method.